### DISCUSSION TITLE
Change the default configuration of the text tool

### DISCUSSION THREAD

**Discussion by DevIA05** [pubpub-zz/ppInk#311] [Q&A]
Hello, 
How i can change the default police of text tool when i use it ? It's on 20 size and bold, i want to set as default 12 size and normal.

**Comment by pubpub-zz** [comment]
in the options :
![image](https://github.com/user-attachments/assets/fbb722c2-b4a3-4e75-9d47-65436d0896b8)

Do not hesitate to star the project if you like it

**Comment by DevIA05** [comment]
Hello, thank for your help.
